python/cudf/cudf/core/dataframe.py

```diff
@@ -6379,8 +6377,10 @@ def _reduce(
         source = self
 
         if axis is None:
+            # TODO(pandas3.0): Remove if/else for just axis = 2
```

Comment:
Rather than a TODO, can we add an assertion here so that we see test failures as soon as we try pandas 3.0?

Reply:
Sure thing. Changed